services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -604,6 +726,16 @@ open class HankeServiceImpl(
         return loggingEntryHolder
     }
 
+    private fun postProcessAndSaveLoggingForRestrictions(loggingEntryHolderForRestrictedActions: YhteystietoLoggingEntryHolder) {
+        loggingEntryHolderForRestrictedActions.applyIPaddresses()
+        loggingEntryHolderForRestrictedActions.saveLogEntries(personalDataAuditLogRepository, personalDataChangeLogRepository)
+        val idList = StringBuilder()
```

Comment:
Method comments for purpose needed. I guess this is buddy of method "postProcessAndSaveLogging" but handles special case for similar?

Reply:
added.